lib/Zureg/Model.hs

```diff
@@ -79,6 +88,122 @@ data Registrant = Registrant
     , rState :: Maybe RegisterState
     } deriving (Show)
 
+instance ToNamedRecord Registrant where
```

Comment:
This code looks all good put I would maybe put all of these instances in a separate `Zureg.Model.Csv` file?